src/sage/rings/function_field/differential.py

```diff
@@ -146,10 +148,11 @@ def _latex_(self):
 
         EXAMPLES::
 
-            sage: K.<x> = FunctionField(GF(4)); _.<Y> = K[]                             # optional - sage.rings.finite_rings
-            sage: L.<y> = K.extension(Y^3 + x + x^3*Y)                                  # optional - sage.rings.finite_rings
-            sage: w = y.differential()                                                  # optional - sage.rings.finite_rings
-            sage: latex(w)                                                              # optional - sage.rings.finite_rings
+            sage: # needs sage.rings.finite_rings
+            sage: K.<x> = FunctionField(GF(4)); _.<Y> = K[]
+            sage: L.<y> = K.extension(Y^3 + x + x^3*Y)
```

Comment:
I don't understand why `# needs sage.rings.function_field` is not needed here

Reply:
Right... It is needed here too. But I currently don't have a working modularized distribution with which I could test either non-prime finite fields or function field extensions. So when doctests are already marked `# needs sage.rings.finite_rings`, the system cannot detect that other `# needs tags` are needed.

I'll check if I can get sagemath-pari in working order to improve this situation.

Reply:
OK. Now my script was able to add this tag automatically.